fillers/rooms.js

```diff
@@ -0,0 +1,121 @@
+const {faker} = require('@faker-js/faker');
+
+function generateRoom() {
+	const roomType = process.env.ROOM_TYPE || 'c';
+
+	let room;
+
+	if (roomType === 'l') {
+		room = generateLivechatRoom();
+
+		return room;
+	}
+
+	const roomName = faker.word.noun() + '_' + faker.number.int(9999);
+
+	room = {
+		_id: faker.git.commitSha({length: 25}),
+		fname: roomName,
+		_updatedAt: {
+			$date: new Date(),
+		},
+		customFields: {},
+		topic: '',
+		broadcast: false,
+		encrypted: false,
+		name: roomName,
+		t: roomType,
+		msgs: 0,
+		usersCount: 1,
+		u: {
+			_id: 'rocketchat.internal.admin.test',
+			username: 'rocketchat.internal.admin.test',
+			name: 'RocketChat Internal Admin Test',
+		},
+		ts: {
+			$date: new Date(),
+		},
+		ro: false,
+		default: false,
+		sysMes: true,
+	};
+
+	return room;
+}
+
+function generateLivechatRoom() {
```

Comment:
Ideally, you would want to have an inquiry associated with this type of rooms. So we should do that as well